lib/build.gradle

```diff
@@ -139,26 +139,26 @@ android {
     externalNativeBuild {
         cmake {
             version "3.10.2"
-            path "${CBL_JAVA_DIR}/CMakeLists.txt"
+            path "${CBL_JAVA_DIR}/lib/CMakeLists.txt"
```

Comment:
I think it might be better to define CBL_JAVA_DIR to include 'lib'.  ... or, if necessary, to define a new var with the lib in it...

Reply:
I will make the change.